style.css

```diff
@@ -1,123 +1,103 @@
-*{
+* {
 	box-sizing: border-box; 
 }
-html{
+
+html {
 	font-size: 20px; 
 }
+
 body {
 	background-color:#771668;
 	width: 100%;
-	margin-top:80px;
-	margin-left:0px;
-	margin-right:0px;
-	margin-bottom:0px;
+	margin-top: 200px;
+	margin-left: 0px;
+	margin-right: 0px;
+	margin-bottom: 0px;
+}
+
+a, p, h1, h2, h3, h4, h5, h6 {
+	color: white;
 }
 
-header{
+header {
 	position: fixed;
-	top:0px;
-	left:0px;
+	top: 0px;
+	left: 0px;
 	background-color: #2d094f;
-	width:100%;
+	width: 100%;
 	padding: 10px;
-	border:3px solid black;
-	z-index:2;
+	border: 3px solid black;
+	z-index: 1;
 }
 
-.nav {
+.id-section-bar {
```

Comment:
as discussed irl, this class is actually not needed because you can just use the header tag instead